lib/features/vpn/vpn_bandwidth.dart

```diff
@@ -8,13 +8,8 @@ class VPNBandwidth extends StatelessWidget {
   Widget build(BuildContext context) {
     return sessionModel
         .bandwidth((BuildContext context, Bandwidth bandwidth, Widget? child) {
-      // User does not have bandwidth cap off
-      if (bandwidth.remaining > 0 || isDesktop()) {
```

Comment:
There was an issue here, If use have not used any of the data here then it shows the progress bar as complete.